features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/public/js/hello.js

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script>
+function myFunction() {
+  document.getElementById("demo").innerHTML = "Paragraph changed.";
+}
+</script>
+</head>
+<body>
+
+<h2>JavaScript in Head</h2>
+
+<p id="demo">A Paragraph.</p>
+
+<button type="button" onclick="myFunction()">Try it</button>
+
+</body>
+</html>
```

Comment:
Add newline at the end of the file.

Reply:
This was a sample file, this file is no longer in the repo.